public/pages/DetectorDetail/components/DetectorControls/DetectorControls.tsx

```diff
@@ -55,6 +55,7 @@ export const DetectorControls = (props: DetectorControls) => {
               key="editDetector"
               data-test-subj="editDetectorSettingsItem"
               onClick={props.onEditDetector}
+              size="s"
```

Comment:
Can you make the `EuiContextMenuPanel` small instead? may just make it easier to add stuff in the future (not sure if this is possible)

Reply:
Because the EuiContextMenu items weren't passed in as items, setting the size attribute for EuiContextMenuPanel didn't change the size of the context menu